README.md

```diff
@@ -63,3 +63,39 @@ Override these rules in `tslint.json`:
 }
 ```
 
+## Inline Linting in Text Editors
```

Comment:
Yah, my bad. 
Should've done a PR like this. 

![screen shot 2017-01-31 at 9 36 00 am](https://cloud.githubusercontent.com/assets/6130700/22469084/bcd8a53e-e798-11e6-8e82-2eb61cbf1f6f.png)